interop/tapir/src/main/scala/caliban/interop/tapir/TapirAdapter.scala

```diff
@@ -199,24 +200,27 @@ object TapirAdapter {
   ) =
     Left(response.toResponseValue(keepDataOnErrors, excludeExtensions))
 
-  def convertHttpEndpointToFuture[R](
-    endpoint: ServerEndpoint[ZioStreams, RIO[R, *]]
-  )(implicit runtime: Runtime[R]): ServerEndpoint[ZioStreams, Future] =
-    ServerEndpoint[
-      endpoint.SECURITY_INPUT,
-      endpoint.PRINCIPAL,
-      endpoint.INPUT,
-      endpoint.ERROR_OUTPUT,
-      endpoint.OUTPUT,
-      ZioStreams,
-      Future
-    ](
+  private val rightUnit: Right[Nothing, Unit] = Right(())
+
+  def convertHttpEndpointToFuture[R, BS, S, I](
```

Comment:
As above, can we make this package-private?

Reply:
Yeah I think you'd always use `serverEndpointsFuture`/`serverEndpointsIdentity` rather than these.